src/components/NewLocationPage/NotesBlock/NotesBlock.style.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+export const TextContainer = styled.div`
+  display: inline-block;
+  flex-direction: column;
+  font-weight: bold;
+`;
+
+export const TextComponent = styled.div`
+  margin-top: 10px;
+  margin-bottom: 10px;
+  font-weight: normal;
+  color: #4f4f4f;
```

Comment:
this color is in `COLOR_MAP`. rather than hardcoding the hex code, you can import `COLOR_MAP` and use `COLOR_MAP.GREY_4`